### PR TITLE
Doc fixes + time.Now() bug.

### PR DESCRIPTION
**Pull request by twifkak**
- Fix several bugs in the documentation.
- Move bin/*.go into the more conventional cmd/ directories, so that
  they integrate well with `go get`.
- Remove GoogleAPIKey until the Transformer API lands.
- Fix time.Now() bug.
- Add some TODOs. ;}

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -4,18 +4,18 @@
 [[projects]]
   branch = "master"
   name = "github.com/WICG/webpackage"
-  packages = [
-    "go/signedexchange/cbor",
-    "go/signedexchange/certurl",
-    "go/signedexchange/mice"
-  ]
+  packages = ["go/signedexchange/certurl"]
```
Comment (nyaxt) [inline Gopkg.lock]: You might want to have this reference github.com/nyaxt/webpackage too in the near future. The WICG repo will be updated to use the new certURL format.

File: README.md [hunk 2]
```diff
@@ -126,15 +127,17 @@ frontend as specified above. In addition, it:
 Once you've chosen a setup that meets the above constraints, actual
 configuration is fairly straightforward:
 
-  1. `git clone http://github.com/ampproject/amp-packager && cd amp-packager && go build bin/amppkg.go`
+  1. `git clone http://github.com/ampproject/amp-packager && cd amp-packager && go build cmd/amppkg/main.go`
```
Comment (nyaxt) [inline README.md]: this is now simply
`go get -v http://github.com/ampproject/amp-packager/cmd/amppkg`

File: cmd/amppkg_test_cache/main.go [hunk 3]
```diff
@@ -42,6 +42,7 @@ func main() {
 	})
 	http.HandleFunc("/test.wpk", func(resp http.ResponseWriter, req *http.Request) {
 		resp.Header().Set("Content-Type", "application/signed-exchange;v=b0")
+		resp.Header().Set("Content-Disposition", "attachment")
```
Comment (nyaxt) [inline cmd/amppkg_test_cache/main.go]: Accidental change?

File: README.md [hunk 4]
```diff
@@ -49,11 +50,11 @@ packages and certificates.
 
 #### Packages
 
-For URLs that look like `https://example.com/wpk/url/to/amp.html`, the frontend
+For URLs that look like `https://example.com/htxg/url/to/amp.html`, the frontend
 server must internally reverse-proxy these requests to something like:
 
 ```
-http://packager.internal/priv-amppkg/doc?fetch=https%3A%2F%2example.com%2Furl%2Fto%2Famp.html&sign=https%3A%2F%2Fexample.com%2Furl%2Fto%2Famp.html
+http://packager.internal/priv/doc?fetch=https%3A%2F%2example.com%2Furl%2Fto%2Famp.html&sign=https%3A%2F%2Fexample.com%2Furl%2Fto%2Famp.html
```
Comment (kinu) [inline README.md]: nit: it looks 'F' before the first 'example.com' got dropped

?fetch=https%3A%2F%2example.com  -->
?fetch=https%3A%2F%2Fexample.com

File: Gopkg.lock (hunk 1 above)
Comment (twifkak) [reply Gopkg.lock]: Oops, yes, done.

File: README.md (hunk 2 above)
Comment (twifkak) [reply README.md]: Done, thanks.

File: cmd/amppkg_test_cache/main.go (hunk 3 above)
Comment (twifkak) [reply cmd/amppkg_test_cache/main.go]: Not accidental, but temporary; I wasn't sure the AMP Cache could make this change, but it turned out to be easy. Reverted.

File: README.md (hunk 4 above)
Comment (twifkak) [reply README.md]: Wow, good catch. Fixed.

**Pull request merged**